packages/pds/src/db/migrations/20230406T185855842Z-feed-item-init.ts

```diff
@@ -0,0 +1,123 @@
+import { DynamicModule, Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<Schema>, dialect: Dialect): Promise<void> {
+  const { ref } = db.dynamic
+  await db.schema
+    .createTable('feed_item')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('type', 'varchar', (col) => col.notNull())
+    .addColumn('postUri', 'varchar', (col) => col.notNull())
+    .addColumn('postAuthorDid', 'varchar', (col) => col.notNull())
+    .addColumn('originatorDid', 'varchar', (col) => col.notNull())
+    .addColumn('sortAt', 'varchar', (col) => col.notNull())
+    .execute()
+  // Fill table. This query should be safe to be run any time to update the index.
+  await db
```

Comment:
I'm wondering if we shouldn't just run this whole thing in a separate app migration 🤔